FordLabs/PeopleMover
# 204 announcement banner

## Issue
Resolves 204

## What was done
- Added announcement banner that sources content from flagsmith

## How to test
1. Navigate to page for first time, see that banner is present.
2. Close banner.
3. Refresh page and see that banner is gone.
4. Change banner text in Flagsmith
5. Visit page in a new tab, see that banner has new text.
